tests/apis.js

```diff
@@ -356,17 +575,18 @@ test('APIs: Types', t => {
   t.true(isEqual(apiTypes.keys, keys), 'Keys are as expected');
 });
 
-test('API: All', t => {
-  return api.all({}).then(results => {
+test.serial('API: All', t => {
+  return api.all({}, allTypes).then(results => {
     t.true(results.hasOwnProperty('items'), 'Has Items');
     t.true(results.hasOwnProperty('pages'), 'Has Pagination');
     t.true(results.items.length >= generated, 'Has at least all items in it');
   });
 });
 
-test('API: All with Follow', t => {
+test.serial.skip('API: All with Follow', t => {
```

Comment:
We need this; in the original PR this was broken, so I need tests for it to make sure it works